lib/types.ts

```diff
@@ -2480,3 +2480,18 @@ export type GroupSummaryResponse = {
 export type MembersCountResponse = {
   count: number;
 };
+
+/**
+ * Response body of get bot info.
+ *
+ * @see [Get bot info](https://developers.line.biz/en/reference/messaging-api/#get-bot-info)
+ */
+export type BotInfoResponse = {
+  userId: string;
+  basicId: string;
+  premiumId?: string;
+  displayName: string;
+  pictureUrl: string;
```

Comment:
should be `pictureUrl?`

Reply:
@xingoxu Thanks. Updated and also rebased the branch onto master.